css/style.css

```diff
@@ -0,0 +1,64 @@
+html{
+
+font-size: 62.5%;    
+}
+
+h1{
+    text-align: center;
+    font-size: 4rem;
+    color: rgb(1, 78, 145);
+}
```

Comment:
I suggest that you assign class names and id to html tags as much as possible instead of using tags directly. Using tags repeatedly in CSS not only increases the risk of confusion but also makes the code "not clean". Please refer to this article for more information: https://frontstuff.io/you-need-to-stop-targeting-tags-in-css